common/eth2/src/lighthouse_vc/types.rs

```diff
@@ -31,6 +32,9 @@ pub struct ValidatorRequest {
     pub gas_limit: Option<u64>,
     #[serde(default)]
     #[serde(skip_serializing_if = "Option::is_none")]
+    pub builder_registration_pubkey_override: Option<PublicKey>,
```

Comment:
Is this required for ValidatorRequest?

Reply:
I think it's nice to have here as an optional, because it makes updating the value via our HTTP API possible